app/serializers/project_serializer.rb

```diff
@@ -32,6 +32,7 @@ def self.page(params = {}, scope = nil, context = {})
       elsif params["state"] == "live"
         params["live"] = true
         params.delete("state")
+        scope = Project.where(state: nil)
```

Comment:
Right, but scope isn't defined in the serializer specs. I guess I can just pass it the class to call #where on. Doesn't seem like the default should be nil if it's being altered, what class should scope be?